packages/specs/schema/src/components/default/classMapper.ts

```
@@ -4,49 +4,20 @@ import {JsonEntityStore} from "../../domain/JsonEntityStore.js";
     // avoid infinite calls
     setValue(options, `components.schemas.${name}`, {});
 
-    options.components!.schemas[name] = execMapper(
-      "any",
-      [store.schema],
-      mapGenericsOptions({
-        ...options,
-        root: false
-      })
-    );
+    options.components!.schemas[name] = execMapper("any", [store.schema], {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)